.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: Report an issue about using Custodian
+labels: [kind/bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what happened!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: expected-bahavior
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what you expected to happen.
+      placeholder: What were you expecting to happen?
+      value: "It did this instead of this"
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Cloud Custodian version and dependency information
+      description: Please run `custodian version --debug` and paste the output here
+      render: shell
+  - type: dropdown
+    id: cloud-provider
+    attributes:
+      label: Cloud Provider
+      description: What cloud provider are you using?
+      options:
+        - Amazon Web Services (AWS)
+        - Microsoft Azure
+        - Google Cloud (GCP)
+        - Kubernetes (Alpha)
+        - OpenStack (Alpha)
+    validations:
+      required: true
+  - type: textarea
+    id: policy
+    attributes:
+      label: Policy
+      description: Please copy and paste the policy you're trying to run if applicable. Please exclude any account/sensitive information. This will be automatically formatted into code, so no need for backticks.  
```

Comment:
`This will be automatically formatted into code, so no need for backticks.` --> `Your response will be automatically formatted into YAML, so code-formatting markdown is not necessary.`

(More specific)